### ISSUE TITLE
Download Zip and Build

### ISSUE THREAD

**Issue by kellycoinguy** [xmlunit/xmlunit.net#16]
First all, thanks for your hard work on this project. It sounds great.

When I download the Zip file on a virgin machine, unzip and load the project, it comes up with 61 errors and 7 warnings when built. 

It would be even more awesome to be able to download an install without having to do a build, but if that isn't the case, then it would be good if the build would work on a virgin machine.

**Comment by bodewig** [comment]
What kind of errors are this?  There shouldn't be any (maybe you've forgotten to perfrom the `nuget restore` step - see https://github.com/xmlunit/xmlunit.net#building).  If it'sd not "just run nuget", then please attach the error output here so we can fix them.

nuget packages of XMLUnit will become available once the code is ready for the first alpha release, which shouldn't be too far away (but I know my time constraints too well to promise anything).

If you follow the Appveyor link from the homepage and click through to the latest build, there'll be an "Artifacts" link that allows you to download nuget packages of the current code base.  I'll state this more prominently on start page.

**Comment by bodewig** [comment]
I assume the problem no longer exists.  If you still can't build from a fresh copy, please open a new issue and provide more details about the errors.
